modules/ROOT/pages/prereq-firewalld-forwarding.adoc

```diff
@@ -0,0 +1,46 @@
+= Enable Forwarding When Using firewalld
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+`firewalld` is an iptables controller that defines rules for persistent network traffic.
+If you are using `firewalld` with a Red Hat Enterprise Linux (RHEL) 7.3 operating system, you must enable forwarding on the docker0 device.
```

Comment:
RTF includes support for RHEL starting at v7.4 (I think). Perhaps we could say 7.x

Reply:
Got it - thank you!